packages/openapi-generator/src/generator.spec.ts

```diff
@@ -6,17 +6,24 @@ import { getSdkVersion } from '@sap-cloud-sdk/generator-common';
 import { emptyDocument } from '../test/test-util';
 import { generate, getInputFilePaths } from './generator';
 
+jest.mock('../../generator-common', () => {
+  const actual = jest.requireActual('../../generator-common');
+  return { ...actual, getSdkVersion: async () => '1.2.3' };
+});
+
 const { readFile } = promises;
 
-// FIXME: These tests are dangerous, because they operate on local data, that has to be generated and does not reside in the package directory, which should not be the case for unit tests.
-// As soon as we have mocking in place this should be exchanged.
 describe('generator', () => {
   const testServicePath = resolve(
     __dirname,
     '../../../test-packages/test-services/openapi/test-service'
   );
 
-  xit('getSdkVersion returns a valid stable version', async () => {
+  afterAll(() => {
+    mock.restore();
+  });
+
+  it('getSdkVersion returns a valid stable version', async () => {
```

Comment:
I guess we can remove this test from here as it's moved to the common.
Also, it's mocked in this file.